src/tunnel.js

```diff
@@ -1,49 +1,79 @@
-var ngrok = require('screener-ngrok');
+var ngrokLauncher = require('screener-ngrok');
 var Promise = require('bluebird');
 var url = require('url');
+var sauceConnectLauncher = require('sauce-connect-launcher');
+var sauceConnection;
 
-exports.connect = function(host, token, tries = 0) {
-  if (!token) {
-    return Promise.reject(new Error('No Tunnel Token'));
+exports.connect = function({ ngrok, sauce }, tries = 0) {
+  if (sauce) {
```

Comment:
Hi, I was wondering about mixing function and fat arrows, should we settle on one approach going forward.

Reply:
Yeah, good point, it's kind of the code style guid, how do you think @loyalchow

Reply:
@xwa130saucer Let's use arrows going forward. But let's not update them all now - we can do so in a separate ticket. Also, we need to update the node requirements in package.json to node >= 6

Reply:
got it, and update the `engine >= 4`